Source/WebCore/crypto/gcrypt/CryptoAlgorithmEd25519GCrypt.cpp

```diff
@@ -105,8 +105,10 @@ static ExceptionOr<bool> verifyEd25519(const Vector<uint8_t>& key, size_t keyLen
 
     // Construct the sig-val s-expression, extracting the r and s components from the signature vector.
     PAL::GCrypt::Handle<gcry_sexp_t> signatureSexp;
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
```

Comment:
@geoffreygaren adopted a specific format for pragmas that are port-specific:
https://commits.webkit.org/286279@main

It would be nice to follow it in this patch. Basically:
```
WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GTK/WPE port
```